code/modules/awaymissions/maploader/reader.dm

```diff
@@ -323,20 +340,20 @@ var/list/map_dimension_cache = list()
 			var/image/new_underlay = image(icon = null) //Because just image() doesn't work, and neither does image(appearance=...)
 
 			new_underlay.appearance = initial(underlying_turf.appearance)
-			T.underlays.Add(new_underlay)
+			if(T)
```

Comment:
Under what circumstances do you expect `T` to be null here? And if it can indeed be null, why bother doing all the earlier work of creating an image and setting its appearance only to then forget about it?
I have the feeling you got a runtime error here and just put a check for null and called it a day.

Reply:
Since fixing this seems to be outside the scope of this PR and I'm not sure what the cause or solution is, should I revert it?